code/mob/living/critter/ai/sawflyai.dm

```diff
@@ -119,24 +122,23 @@
 	..()
 
 /datum/aiTask/sequence/goalbased/sawfly_chase_n_stab/get_targets()
-	. = list()
+	. = list() // don't ask why the list is simply a period
```

Comment:
```suggestion
	. = list()
```
`.` is standard and documented, not really anything all that unusual https://www.byond.com/docs/ref/#/proc/%2e